src/client/components/MyNotifications/Notification.tsx

```diff
@@ -76,7 +76,9 @@ const Notification = ({
           onClick={() => toggleCollapsed(!isCollapsed)}
         >
           <Heading className={styles.NotificationHeader} level={4}>
-            {notification.title}
+            {notification.isTip
+              ? `Tip: ${notification.title}`
+              : notification.title}
```

Comment:
In `tips-content` staan alle tip titels al prexied met `Tip:` Is het een front-end verantwoordelijkheid om hier `Tip:` neer te zetten?

Reply:
oke, ga ik uitzoeken